src/man/sssd.conf.5.xml

```diff
@@ -202,6 +202,11 @@
                             <para>
                                 Comma separated list of services that are
                                 started when sssd itself starts.
+                                <phrase condition="have_systemd">
+                                    or SSSD 1.15.0 or later the services' list is optional
```

Comment:
Do you think the version number is needed? I'm not convinced one way or another, but I don't think we will ever back port these patches, so maye we could just omit the version number?